langlib/langlib-test/src/test/resources/test-src/tablelib_test.bal

```diff
@@ -282,8 +282,20 @@ function removeIfHasKey() returns boolean {
     return removedPerson1 == () && removedPerson2?.name == "Chiran";
 }
 
-function testHasKey() returns boolean {
-    return tab.hasKey("Mohan");
+function testHasKey() {
```

Comment:
lets add more tests for union types also primitive types and container key types